core/src/main/com/ruslan/entity/book/Book.java

```diff
@@ -1,124 +1,52 @@
 package com.ruslan.entity.book;
 
+
+import com.ruslan.entity.BaseEntity;
+import com.ruslan.entity.order.Order;
+import com.ruslan.entity.request.Request;
+import jakarta.persistence.*;
+import lombok.*;
+
 import java.io.Serializable;
 import java.time.LocalDate;
-import java.util.Objects;
+import java.util.List;
+
+@Entity
+@Table(name = "book", schema = "bookstore")
+@Data
```

Comment:
please read what @data  doing and why it's not recommended to use it with entities

Reply:
Thank you, I replaced @Data